core/src/main/resources/lib/layout/dropdowns/custom.jelly

```diff
@@ -31,7 +31,7 @@ THE SOFTWARE.
 
         <![CDATA[
             <dd:custom>
-              <img src="jenkins.svg" />
+              <img src="jenkins.svg" alt="null"/>
```

Comment:
```suggestion
              <img src="jenkins.svg" alt="Jenkins logo"/>
```

Reply:
My recommendation is wrong here because this is an example embedded in documentation so I think that it is better to use `alt=""` than try to describe the contents of the image.